### ISSUE TITLE
Failed to execute command. Exception:

### ISSUE THREAD

**Issue by Zytinx** [Z0rdak/Yet-Another-World-Protector#92]
Cant set region in game as command says error has occured trying to execute

```
21.07 12:40:26 [Server] Server thread/ERROR Failed to execute command. Exception: 
21.07 12:40:26 [Server] INFO java.lang.NullPointerException: Cannot invoke "de.z0rdak.yawp.core.region.IMarkableRegion.hasOwner(java.util.UUID)" because "region" is null
21.07 12:40:26 [Server] INFO at de.z0rdak.yawp.handler.CommandInterceptor.handleRegionCmdExecution(CommandInterceptor.java:85) ~[yawp-1.18.2-0.0.2.9-beta1-forge.jar%23284!/:0.0.2.9-beta1] {re:classloading,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at de.z0rdak.yawp.handler.CommandInterceptor.handleModCommandPermission(CommandInterceptor.java:55) ~[yawp-1.18.2-0.0.2.9-beta1-forge.jar%23284!/:0.0.2.9-beta1] {re:classloading,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraftforge.eventbus.ASMEventHandler_1408_CommandInterceptor_handleModCommandPermission_CommandEvent.invoke(.dynamic) ~[?:?] {}
21.07 12:40:26 [Server] INFO at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?] {}
21.07 12:40:26 [Server] INFO at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?] {}
21.07 12:40:26 [Server] INFO at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?] {}
21.07 12:40:26 [Server] INFO at net.minecraft.commands.Commands.m_82117_(Commands.java:226) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,re:classloading,pl:rei_plugin_compatibilities:B,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.network.ServerGamePacketListenerImpl.m_9957_(ServerGamePacketListenerImpl.java:1150) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:wildbackport-common.mixins.json:network.ServerGamePacketListenerImplMixin,pl:mixin:APP:braincell.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:APP:timeout_fixes.mixins.json:ServerPlayNetHandlerKeepAliveMixin,pl:mixin:APP:servercore.common.mixins.json:features.misc.ServerGamePacketListenerImplMixin,pl:mixin:APP:servercore.common.mixins.json:optimizations.sync_loads.ServerGamePacketListenerImplMixin,pl:mixin:APP:vanishmod.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:APP:dragonmounts.mixins.json:EnsureSafeFlyingVehicleMixin,pl:mixin:APP:toofast.mixins.json:MixinServerNetPlayHandler,pl:mixin:APP:mixins.dcintegration.json:MixinNetHandlerPlayServer,pl:mixin:APP:dannys_expansion.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.network.ServerGamePacketListenerImpl.m_143628_(ServerGamePacketListenerImpl.java:1129) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:wildbackport-common.mixins.json:network.ServerGamePacketListenerImplMixin,pl:mixin:APP:braincell.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:APP:timeout_fixes.mixins.json:ServerPlayNetHandlerKeepAliveMixin,pl:mixin:APP:servercore.common.mixins.json:features.misc.ServerGamePacketListenerImplMixin,pl:mixin:APP:servercore.common.mixins.json:optimizations.sync_loads.ServerGamePacketListenerImplMixin,pl:mixin:APP:vanishmod.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:APP:dragonmounts.mixins.json:EnsureSafeFlyingVehicleMixin,pl:mixin:APP:toofast.mixins.json:MixinServerNetPlayHandler,pl:mixin:APP:mixins.dcintegration.json:MixinNetHandlerPlayServer,pl:mixin:APP:dannys_expansion.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.network.ServerGamePacketListenerImpl.m_7388_(ServerGamePacketListenerImpl.java:1115) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:wildbackport-common.mixins.json:network.ServerGamePacketListenerImplMixin,pl:mixin:APP:braincell.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:APP:timeout_fixes.mixins.json:ServerPlayNetHandlerKeepAliveMixin,pl:mixin:APP:servercore.common.mixins.json:features.misc.ServerGamePacketListenerImplMixin,pl:mixin:APP:servercore.common.mixins.json:optimizations.sync_loads.ServerGamePacketListenerImplMixin,pl:mixin:APP:vanishmod.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:APP:dragonmounts.mixins.json:EnsureSafeFlyingVehicleMixin,pl:mixin:APP:toofast.mixins.json:MixinServerNetPlayHandler,pl:mixin:APP:mixins.dcintegration.json:MixinNetHandlerPlayServer,pl:mixin:APP:dannys_expansion.mixins.json:ServerGamePacketListenerImplMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.network.protocol.game.ServerboundChatPacket.m_5797_(ServerboundChatPacket.java:30) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:classloading,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.network.protocol.game.ServerboundChatPacket.m_5797_(ServerboundChatPacket.java:7) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:classloading,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.network.protocol.PacketUtils.m_131356_(PacketUtils.java:22) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:classloading,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:classloading,pl:rei_plugin_compatibilities:B,re:mixin}
21.07 12:40:26 [Server] INFO at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:157) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,re:computing_frames,pl:rei_plugin_compatibilities:B,re:classloading,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:799) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:flan.mixins.json:MinecraftServerMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:164) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:flan.mixins.json:MinecraftServerMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:131) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:782) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:flan.mixins.json:MinecraftServerMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:776) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:flan.mixins.json:MinecraftServerMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:140) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:762) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:flan.mixins.json:MinecraftServerMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:689) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:APP:globaldataandresourcepacks.mixins.json:PackConfigurationMixin,pl:mixin:APP:flan.mixins.json:MinecraftServerMixin,pl:mixin:A}
21.07 12:40:26 [Server] INFO at net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261) ~[server-1.18.2-20220404.173914-srg.jar%23297!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:servercore.common.mixins.json:features.misc.MinecraftServerMixin,pl:mixin:APP:otyacraftengine-common.mixins.json:MinecraftServerMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:servercore.common.mixins.json:MinecraftServerMixin,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.js
```

Minecraft version: 1.18.2
YAWP version: 1.18.2-0.0.2.9-beta1-forge.

Edit: Format Stacktrace as code. (Z0rdak), Add MC and YAWP Version

**Comment by Z0rdak** [comment]
Hey there,
thanks for the stacktrace. Please try to provide a little more context though.

What were you trying to execute when this happened?

Best regards

**Comment by Zytinx** [comment]
I was trying to run the command /wp region minecraft:overworld Boss area Cuboid (with the coords here)

But upon running it it would give me that error. It worked perfectly fine before as I have 2 zones that do work currently, but for some reason im unable to make a new one

**Comment by Z0rdak** [comment]
Thanks for the context.

The `area` command to update the area of a Local Region is currently bugged. Sorry about that.

I am currently working on a fix and hope to release it this week. Until then, I am afraid you have to delete and redefine your region to change the area.

**Comment by Z0rdak** [comment]
Fixed in 0.0.2.9-beta3, see #89 

After further investigation I could only replicate this issue when an invalid region name was supplied. Then this error is understandable.
I added an extra check with a more concise error message for this.

If you have further issues with this feel free to open a new issue.
